rules/windows/defense_evasion_delete_volume_usn_journal_with_fsutil.toml

```
@@ -17,14 +17,42 @@ index = ["winlogbeat-*", "logs-endpoint.events.*", "logs-windows.*", "endgame-*"
 language = "eql"
 license = "Elastic License v2"
 name = "Delete Volume USN Journal with Fsutil"
-note = """## Setup
+note = """## Triage and analysis
 
-If enabling an EQL rule on a non-elastic-agent index (such as beats) for versions <8.2, events will not define `event.ingested` and default fallback for EQL rules was not added until 8.2, so you will need to add a custom pipeline to populate `event.ingested` to @timestamp for this rule to work.
+### Investigating Delete Volume USN Journal with Fsutil
+
+The USN Journal (Update Sequence Number Journal) is a feature in the NTFS file system used by Microsoft Windows operating systems to keep track of changes made to files and directories on a disk volume. The journal records metadata about changes such as file creation, deletion, modification, and permission changes. It is used by the operating system for various purposes, including backup and recovery, file indexing, and file replication.
+
+This artifact can provide valuable information in forensic analysis, such as programs executed (prefetch file operations), file modification events in suspicious directories, deleted files, etc. Attackers may delete this artifact in an attempt to cover their tracks, and this rule identifies the usage of the `fsutil.exe` utility to accomplish it.
+
+Remembering that the Elastic Defend integration gives both more visibility and context than the USN journal in the file operations it records.
```

This suggested revision invites users to use the Elastic Defend integration for their change tracking needs. 

```suggestion
Consider using the Elastic Defend integration instead of USN Journal, as the Elastic Defend integration provides more visibility and context in the file operations it records.
```